src/main/java/net/fabricmc/mappingio/MappingReader.java

```diff
@@ -58,7 +58,7 @@ public static MappingFormat detectFormat(Reader reader) throws IOException {
 		int pos = 0;
 		int len;
 
-		// Be careful not to close the reader, thats upto the caller.
+		// Be careful not to close the reader, thats up to the caller.
```

Comment:
```suggestion
		// Be careful not to close the reader, that's up to the caller.
```
Might as well fix it all the way 😛

Reply:
Oops, completely missed it 😄